translations/puddletag_de.ts

```diff
@@ -143,42 +143,42 @@ The &lt;b&gt;Oxygen team&lt;/b&gt; for the Oxygen icons.
     <message>
         <location filename="../puddlestuff/tagsources/acoust_id.py" line="256"/>
         <source>Minimum Score</source>
-        <translation></translation>
+        <translation>Minimale Punktzahl</translation>
     </message>
     <message>
         <location filename="../puddlestuff/tagsources/acoust_id.py" line="306"/>
         <source>Parsing Data</source>
-        <translation></translation>
+        <translation>Daten werden eingelesen</translation>
```

Comment:
Does parsing imply analysis? If "Max Mustermann" is parsed to obtain first and last name, it will split it into "Max" and "Mustermann". The analysis part would be to check if it's a plausible content (and "Mustermann" is most likely fake).

Parsing has no good analogue in German, usually I use it as anglicism. That seems to be legit judging from the Duden (https://www.duden.de/rechtschreibung/parsen).